doc/man3/OSSL_CMP_CTX_new.pod

```diff
@@ -532,7 +532,8 @@ CertRepMessage or Revocation Response or error message, or NULL if unset.
 
 OSSL_CMP_CTX_get_failInfoCode() returns the error code from the failInfo field
 of the last received CertRepMessage or Revocation Response or error message.
-This is a bit field and the flags for it are specified in L<cmp.h>.
+This is a bit field and the flags for it are specified in the header file
+C<cmp.h>.
```

Comment:
That should be `F<< <openssl/cmp.h> >>`

Reply:
Done.  It is nice to see that some practices have improved since this PR was first created.